### DISCUSSION TITLE
How to submit a plugin?

### DISCUSSION THREAD

**Discussion by xigoi** [codicocodes/dotfyle#77] [Q&A] [answered]
I don't find it clear from the website how one can add their plugin to appear on it.

**Comment by chrisgrieser** [comment]
As far as I can tell, the [awesome-neovim](https://github.com/rockerBOO/awesome-neovim/blob/main/README.md) repo is crawled regularly, meaning that you'd add it to that repo

**Comment by codicocodes** [comment, marked as answer]
You can now submit plugins here: https://dotfyle.com/neovim/plugins/add
